src/quo2/foundations/shadows.cljs

```diff
@@ -0,0 +1,70 @@
+(ns quo2.foundations.shadows
+  (:require [quo2.foundations.colors :as colors]
+            [quo2.theme :as theme]))
+
+
+(defn- get-inverted [inverted? number] (if inverted? (* -1 number) number))
```

Comment:
I think this could be easier to read with at least a line break after the square brackets. In general, single-line function definitions should be avoided.

```clojure
(defn- get-inverted [inverted? number]
  (if inverted? (* -1 number) number))
```